docs/resources/oxide_instance.md

```diff
@@ -8,50 +8,57 @@ This resource manages instances.
 
 -> Updates will stop and reboot the instance.
 
--> When setting a boot disk, the boot disk ID should also be included as part of `disk_attachments`.
-
 ## Example Usage
 
-### Basic instance with attached disks and a network interface
+### Instance with user data and an SSH public key
+
+```hcl
+resource "oxide_instance" "example" {
+  project_id       = "c1dee930-a8e4-11ed-afa1-0242ac120002"
+  description      = "Example instance."
+  name             = "myinstance"
+  host_name        = "myhostname"
+  memory           = 10737418240
+  ncpus            = 1
+  disk_attachments = ["611bb17d-6883-45be-b3aa-8a186fdeafe8"]
+  ssh_public_keys  = ["066cab1b-c550-4aea-8a80-8422fd3bfc40"]
+  user_data        = filebase64("path/to/init.sh")
```

Comment:
To show a minimal instance we should remove these two optional settings and add them to another more complex example.